resources/opennebula/opennebula.tf.tmpl

```diff
@@ -24,9 +24,8 @@ resource "opennebula_virtual_machine" "${vm_name}" {
     model = "host-passthrough"
   }
   disk {
-    image_id: ${image_id}
-    size: ${disk_size}
```

Comment:
Why don't we need the `size` parameter anymore?

Reply:
Accepted